scripts/generate_release_notes.py

```diff
@@ -179,7 +180,9 @@ def categorize_entries(
         if "release-highlight" in label_names:
             categories["highlights"].append(entry)
 
-        if "bug" in label_names:
+        if "breaking-change" in label_names:
```

Comment:
We could adopt `breaking`, but just wanted to follow recent PRs.

Reply:
I am okay with either

Reply:
I'll update to just "breaking" because the other tags don't have "-changes" and sync your PR.

Reply:
Updated your PR tag and Notion/info/slack